### DISCUSSION TITLE
Rollup.runFromTrigger() in FFLIB Trigger Handler

### DISCUSSION THREAD

**Discussion by kechetino** [jamessimone/apex-rollup#606] [General]
If I am using the FFLIB Tigger Handler Framework, where do I add the one line of code(Rollup.runFromTrigger()), do I need to add it to each trigger context specific method (onAfterInsert, onAfterUpdate, etc.) or is there some way I can just write it once for all the methods?

Thanks.

**Comment by jamessimone** [comment]
@kechetino I've submitted https://github.com/apex-enterprise-patterns/fflib-apex-common/pull/491 to aid with this. However, I'm converting this to a discussion as it's not an issue with Apex Rollup.

**Reply by kechetino** [reply]
Thanks @jamessimone. Highly appreciated!
